### ISSUE TITLE
text/blob types may only have expression default values

### ISSUE THREAD

**Issue by ericmock** [dolthub/dolt#3388]
mysqldump spit this out:

CREATE TABLE `ICD_codes` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `code` text DEFAULT NULL,
  `title` blob DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=MyISAM AUTO_INCREMENT=11139 DEFAULT CHARSET=latin1

Dolt gave the error in the title.  Seems like there's quite a bit of discussion about MySQL and default values for text and blob types.

**Comment by timsehn** [comment]
Repro.

```
test-default $ dolt init
Successfully initialized dolt data repository.
test-default $ emacs table.sql
test-default $ dolt sql < table.sql 
error on line 1 for query CREATE TABLE ICD_codes (
  id int(11) unsigned NOT NULL AUTO_INCREMENT,
  code text DEFAULT NULL,
  title blob DEFAULT NULL,
  PRIMARY KEY (id)
) ENGINE=MyISAM AUTO_INCREMENT=11139 DEFAULT CHARSET=latin1
: text/blob types may only have expression default values
text/blob types may only have expression default values
```

**Comment by fulghum** [comment]
Thanks for reporting this so we can get this fixed. 

As a short term workaround in case you're blocked on getting that dump loaded into Dolt, you can wrap NULL in parens in the create table statement or you can just remove the `DEFAULT NULL` from the column definition in the dump file. 

We'll dig into what's going on and get this valid syntax working.

**Comment by ericmock** [comment]
The defaults weren't really necessary, so I just axed them.

**Comment by fulghum** [comment]
Thanks again @ericmock . We got the fix for this merged into main and it'll go out in a release later this week. 

Let us know if you hit any other problems or have any questions and we'll be happy to help.

**Comment by bpf120** [comment]
@ericmock Thanks for reporting. We'd love to learn about your use case too.  You can email me (brianf@dolthub.com) or swing by our Discord if you'd like to share.  https://discord.com/invite/RFwfYpu
